tests/test_pyglmnet.py

```diff
@@ -388,3 +385,61 @@ def test_random_state_consistency():
 
     assert_array_equal(ypred_a, ypred_b)
     assert_array_equal(ypred_b, ypred_c)
+
+
+@pytest.mark.parametrize("distr", ALLOWED_DISTRS)
+def test_simulate_glm(distr):
+    """
+    Test that every generative model can be simulated from.
+    """
```

Comment:
numpy doctring conventions dictate that L392-L394 should be in one line

Reply:
Done.